src/teaser.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title> Vulpem Ventures </title>
+  <meta charset="UTF-8">
+  <meta name="description" content="Technology R&D lab focused on cybersecurity of digital assets">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!--Favicon-->
+  <link rel="apple-touch-icon" sizes="180x180" href="./images/favicon/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="./images/favicon/favicon-16x16.png">
+  <link rel="manifest" href="./images/favicon/site.webmanifest">
+  <link rel="mask-icon" href="./images/favicon/safari-pinned-tab.svg" color="#171615">
+  <meta name="msapplication-TileColor" content="#181f1f">
+  <meta name="theme-color" content="#ffffff">
+  <!-- build:css css/combined.css -->
+  <link rel="stylesheet" href="./css/vendors.css">
+  <link rel="stylesheet" href="./css/style.css">
+  <!-- endbuild -->
+</head>
+
+<body>
+<div class="teaser">
+  <div class="container teaser-container">
+    <div class="teaser-header">
+      <a class="logo-wrap" href="/">
+        <img class="logo-image" src="./images/logo/100x86.png">
+        <span class="logo-text">Vulpem  </span>
+      </a>
+    </div>
+    <div class="columns">
+      <div class="column">
+        <img class="teaser-mobile-img" src="./images/iphone-img.png">
+      </div>
+      <div class="column teaser-content-column">
+        <h1 class="teaser-title teaser-title-lg">VULPEM Alpha Mobile App</h1>
+        <h2 class="teaser-subtitle">It’s all about transmit fear <br> of loosing your keys if you do by yourself,<br> but at the same time our solution it's <br> trustless (we cannot keep or hold your money, <br> you are in 100% of control)</h2>
+        <form class="form-primary teaser-form" action="">
+          <input class="input input-primary teaser-form-input" placeholder="Your email" type="text">
+          <button class="button btn-action">Send</button>
+        </form>
+        <div class="features">
+          <div class="column features-item">
+            <img src="./images/trustless-icon.png" class="features-item-image">
+            <div class="features-item-text">Trustles</div>
+          </div>
+          <div class="column features-item">
+            <img src="./images/inheritance-solution-icon.png" class="features-item-image">
+            <div class="features-item-text">Inheritance Solution</div>
```

Comment:
the fontSize of feature's text should be bigger

Reply:
Hi Marco. I add changes, please review it.